wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -837,6 +838,17 @@ flushWalletStorage = modify flushDo
                             , _wiIsReady   = False
                             }
 
+-- | Unsafe address conversion for use in migration. This will throw an error if
+--   the address cannot be migrated.
+unsafeCIdToAddress :: WebTypes.CId WebTypes.Addr -> Address
+unsafeCIdToAddress cId = case WebTypes.cIdToAddress cId of
+    Left err -> error $ "unsafeCIdToAddress: " <> err
+    Right x  -> x
```

Comment:
If anyone wants to make the patch even smaller, we don't need to move this function around in the file. Not important of course.